.travis.yml

```
@@ -1,25 +1,21 @@
 language: scala
+
+jdk: oraclejdk8
+
 matrix:
-  include:
-    - scala: 2.11.12
-      jdk: oraclejdk8
-    - scala: 2.12.6
-      jdk: oraclejdk8
+  fast_finish: true
```

this will make sure that if one of the steps fail the build will fail fast.